### PR TITLE
Basic gametype statistics logging.

### PR DESCRIPTION
**Pull request by oisin100**
Fixes issue #1152 
### Intent of your Pull Request

Read #1152

and take a look at the [site](http://yogstats.oisin100.net)

Im terrible at HTML and CSS btw.
#### Changelog

:cl:
rscadd: Adds basic statistics gathering for gamemodes, Temporary site can be found at http://yogstats.oisin100.net
/:cl:

### CODE REVIEW

**Comment by oisin100** [comment]
Currently only the Gamemode section has full functionality, Functionality for the other features will be added in other PR's

Stats are only sent to Yogstats at round end.

File: code/modules/mob/living/carbon/death.dm [hunk 1]
```diff
@@ -2,4 +2,5 @@
 	silent = 0
 	med_hud_set_health()
 	med_hud_set_status()
+	ticker.total_deaths++ //We only want to log carbon animals deaths.
 	..(gibbed)
```
Comment (Cruix) [inline code/modules/mob/living/carbon/death.dm:7]: You should probably check for a key so a mad xenobiologist or botanist can't inflate the death counter by spamming monkey cubes.

**Comment by Cruix** [comment]
Looks great Oisin good job :+1:

File: code/modules/mob/living/carbon/death.dm (hunk 1 above)
Comment (oisin100) [inline code/modules/mob/living/carbon/death.dm:7]: Mm, Good idea!

**Comment by FluffySurvivor** [comment]
why is it not merged

**Comment by oisin100** [comment]
> why is it not merged

Waiting for council vote.

**Pull request merged**